react-search/my-app/src/SearchableList.tsx

```diff
@@ -0,0 +1,26 @@
+import { useState } from 'react';
+import './SearchableList.css';
+import { SearchBar } from './SearchBar';
+import { Items } from './Items';
+type Props = {
+  quotes: string[];
+};
+export function SearchableList({ quotes }: Props) {
+  const [searchValue, setValue] = useState('');
+  const [filteredQuotes, setFilteredQuotes] = useState(quotes);
```

Comment:
This works with it being state, but it also works if you calculate `filteredQuotes` as a const instead of as state.